app/javascript/admin/controllers/ahoy_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from 'stimulus';
+import ahoy from 'ahoy.js';
+
+ahoy.configure({
+  cookies: false,
+  trackVisits: false,
+});
+
+// eslint-disable-next-line no-restricted-syntax
+export default class AhoyController extends Controller {
+  trackOverviewLink(event) {
+    event.preventDefault();
+    let properties = {
+      action: event.type,
+      target: event.target.toString(),
+    };
+    ahoy.track('Overview Link Clicked', properties);
```

Comment:
```suggestion
    ahoy.track('Admin Overview Link Clicked', properties);
```
Might just make it a tad bit easier down the road when we want to parse all events by the admin portal (as an example)